http-netty/src/main/java/io/micronaut/http/netty/reactive/HandlerPublisher.java

```diff
@@ -259,13 +259,13 @@ public void channelRegistered(ChannelHandlerContext ctx) {
     private void provideChannelContext(ChannelHandlerContext ctx) {
         switch (state) {
             case NO_SUBSCRIBER_OR_CONTEXT:
-                verifyRegisteredWithRightExecutor(ctx);
+                verifyRegisteredWithRightExecutor();
```

Comment:
I don't think this change is correct. The instance variable isn't set until after the method is called

Reply:
`ctx` was not used in `verifyRegisteredWithRightExecutor`

Reply:
Ah I see. Github collapsed some code and it looked like it was. Thanks